modules/ROOT/pages/_partials/proc_self-signed-certs.adoc

```diff
@@ -40,36 +40,54 @@ The resulting file is located in the `/tmp` directory.
 
 [tabs]
 ====
-// tag::excludeDownstream[]
 Android::
```

Comment:
@finp / @wei-lee - the (tag::excludeDownstream[]) have been removed from here. I'm not sure if this is in-line with what we are GA'ing in September...it could be, just calling it out for your attention.